components/provider_cldr/Cargo.toml

```diff
@@ -32,11 +32,15 @@ icu_locid = { version = "0.1", path = "../locid" }
 icu_plurals = { version = "0.1", path = "../plurals" }
 icu_datetime = { version = "0.1", path = "../datetime" }
 icu_locale_canonicalizer = { version = "0.1", path = "../locale_canonicalizer" }
+icu_decimal = { version = "0.1", path = "../decimal" }
+itertools = "0.10"
 json = "0.12"
 serde = { version = "1.0", features = ["derive"] }
+serde-aux = "2.1.1"
```

Comment:
Suggestion (optional): After reading through the code, I'm a little nervous with adding this dependency. There is a trade off on adding and maintaining dependencies, and the value you get from them. To me, this feels like it's on the wrong side of the trade off. It would not be too much work to get around not using it.

I won't block if you want to keep it, but I would suggest not adding.

Reply:
I added it because:

1. It was an expedient way to solve a problem I had (the `"deserialize_number_from_string"` attribute)
2. cldr_json is an offline tool and has a lower bar for what kinds of dependencies we can add
3. The dependency can be removed in the future if it becomes incompatible
4. The author of the crate owns several other Rust crates and appears to be motivated to maintain it

But I agree that we shouldn't generally add dependencies willy-nilly, and I expected this to be a question during the review. Let's discuss tomorrow.

Reply:
> But I agree that we shouldn't generally add dependencies willy-nilly, and I expected this to be a question during the review. Let's discuss tomorrow

I can leave it up to your judgement, but I like putting a bit a bit of friction in place for any new dependency added to stop and make sure we really need it.

Reply:
I made a note to have this discussion with the larger group in the context of #574.  For now I'll just keep this the way it is.